bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/internal/firmware/FirmwareUpdateServiceTest.java

```diff
@@ -68,6 +76,7 @@
  * @author Thomas Höfer - Initial contribution
  * @author Simon Kaufmann - converted to standalone Java tests
  * @author Dimitar Ivanov - added a test for valid cancel execution during firmware update
+ * @author Dimitar Ivanov - replaced Firmware UID with thing UID and firmware version
```

Comment:
I am not sure - is it common that we add the author twice?

Reply:
I checked and you are right - the contributions are comma separated instead of on new line. I will change this.